scripts/osfstorage/usage_audit.py

```diff
@@ -0,0 +1,99 @@
+"""File: usage_audit.py
+Find all users and projects where their total usage (current file + deleted files) is >= the set limit
+Projects or users can have their GUID whitelisted via `usage_audit whitelist [GUID ...]`
+User usage is defined as the total usage of all projects they have > READ access on
```

Comment:
Nope, the whitelist only makes the targeted GUID exempt.
I think the situation you laid out makes more sense than the current behavior, changing now.